app/models/helper.py

```diff
@@ -0,0 +1,14 @@
+from .planet import Planet
+from flask import abort, make_response
+
+def validate_id(planet_id):
+    try:
+        planet_id = int(planet_id)
+    except:
+        return abort(make_response({"message": f"Planet {planet_id} is not valid"}, 400))
+    
+    planet = Planet.query.get(planet_id)
+    if not planet:
+        return abort(make_response({"message": f"Planet {planet_id} does not exist"}, 404))
+
+    return planet
```

Comment:
Nice work pulling this into a helper to keep your routes short and sweet. In the future, you might need to make 'helper.py' more specific. Think about Task List, you have Goal and Task, you may need helpers for each. If you refactor helper methods into a helper file then you could have goal_helper.py and task_helper.py